src/main/webapp/css/main.css

```diff
@@ -823,6 +839,10 @@ form.button {
 }
 
 /* task sheet */
+.sheetholder {
```

Comment:
This breaks the timesheet page, it should be only applied to the tasksheet table.